app/lnd/lib/neutrino.js

```diff
@@ -79,11 +80,23 @@ class Neutrino extends EventEmitter {
       if (!this.state.chainSyncStarted) {
         const match = line.match(/Syncing to block height (\d+)/)
         if (match) {
-          const height = match[1]
           this.state.chainSyncStarted = true
-          this.state.targetBlockHeight = height
           this.emit('chain-sync-started')
-          this.emit('got-final-block-height', height)
```

Comment:
Would it work to issue this then re-issue it? Would make for more immediate feedback if it works.

Reply:
Yes, nice idea. I've updated this so that we issue the btcd height as soon as we have it, and then follow up with the highest of either the btcd height or the fetched height once we have that